### DISCUSSION TITLE
Show ProjectM Visualizer instead of the image

### DISCUSSION THREAD

**Discussion by rusagent** [TT-ReBORN/Georgia-ReBORN#7] [Q&A] [answered]
The title says it all. How could that be achieved?

**Comment by TT-ReBORN** [comment, marked as answer]
Hi, 

this does not seem to be possible because Georgia-ReBORN is one full single Spider Monkey Panel. You can ask the developer djdron if this plugin can be called via javascript and how to implement it. Here is the link: https://hydrogenaud.io/index.php?topic=120115.0

Alternatively you can run it in a new window via the player top menu: 
View -> Visualization -> projectM

-TT
